yarn-error.log

```diff
@@ -1,14 +1,14 @@
 Arguments: 
```

Comment:
Please commit the removal of  this file from the repository and add it to .gitignore
Something like this:
```
git rm yarn-error.log
```
add this line to .gitignore
```
yarn-error.log
```
and commit these two changes (removal and change to .gitignore).